queries/git_config/injections.scm

```diff
@@ -6,3 +6,26 @@
   value: (string) @injection.content)
   (#any-of? @_name "cmd" "command" "textconv")
   (#set! injection.language "bash"))
+
+(section
+  (section_header
+    (section_name) @_section)
+  (variable
+    value: (string) @injection.content)
+  (#eq? @_section "alias")
+  (#lua-match? @injection.content "^!")
+  (#offset! @injection.content 0 1 0 0)
+  (#set! injection.language "bash"))
+
+(section
+  (section_header
+    (section_name) @_section)
+  (variable
+    value:
+      (string
+        "\""
+        "\"") @injection.content)
+  (#eq? @_section "alias")
+  (#lua-match? @injection.content "^\"!")
```

Comment:
What about single quotes?

Reply:
I think single quotes may not be an option, I tried them out in my config and single quotes cause errors where double quotes do not. Same with adding single quotes around an unquoted command

Reply:
Tree-sitter errors or git errors?